pkg/kubectl/cmd/wait/wait.go

```diff
@@ -449,3 +451,7 @@ func (w ConditionalWait) isConditionMet(event watch.Event) (bool, error) {
 	obj := event.Object.(*unstructured.Unstructured)
 	return w.checkCondition(obj)
 }
+
+func extendErrWaitTimeout(err error, resourceType, resource string) error {
```

Comment:
Why not just pass `info *resource.Info` ?

Reply:
Doesn't wanted to make it depended on concrete type/struct.

Reply:
Doesn't wanted to make it depended on concrete type/struct.

Reply:
But if you see it as a critical point, i will easily switch it.